scripts/cat/sprites.py

```diff
@@ -81,26 +81,22 @@ def make_group(self,
         """
 
         # making the group
-        new_group = pygame.Surface(
-            (self.size * sprites_x, self.size * sprites_y),
-            pygame.HWSURFACE | pygame.SRCALPHA)
-        new_group.blit(
-            self.spritesheets[spritesheet], (0, 0),
-            (pos[0] * sprites_x * self.size, pos[1] * sprites_y * self.size,
-             (pos[0] + sprites_x) * self.size,
-             (pos[1] + sprites_y) * self.size))
+        new_group = pygame.Surface.subsurface(self.spritesheets[spritesheet],
+                                              pos[0] * sprites_x * self.size,
+                                              pos[1] * sprites_y * self.size,
+                                              self.size * sprites_x,
+                                              self.size * sprites_y)
 
         self.groups[name] = new_group
```

Comment:
Are you able to remove self.groups entirely? As far as I remember, it not used outside of this function.